src/lib/dao.js

```diff
@@ -572,6 +573,8 @@ foam.CLASS({
   methods: [
     function put(o) {
       this.a.push(o);
+    },
+    function eof() {
```

Comment:
Ah, yep, that's true. I added `eof()` before realizing that ti didn't implement `Sink`.

Removed.